packages/runtime/src/runtime.ts

```diff
@@ -234,6 +233,25 @@ export class Runtime {
     return this.assertAssetDefined(assetId, assetDef);
   }
 
+  /**
+   * Queries asset id by asset name from global state.
+   * Returns undefined if asset is not found.
+   * @param name Asset name
+   */
+  getAssetIdFromName (name: string): number | undefined {
+    return this.store.assetNameId.get(name);
+  }
+
+  /**
+   * Queries app id by app name from global state.
+   * Returns undefined if app is not found.
+   * @param approval
+   * @param clear
+   */
+  getAppIdFromName (approval: string, clear: string): number | undefined {
+    return this.store.appNameId.get(approval + "-" + clear);
+  }
```

Comment:
Can we return app/asset info here? As while loading from checkpoints we do something similar, so user can have same experience in tests as well. What do you think?
Note: i think app/asset info has their id's (along with other metadata)

Reply:
yes